### PR TITLE
Docs++ & idea for better embed handling

### PR DESCRIPTION
**Pull request by AlexBlandin**
So, ideally, we could add an `embed` bool and use that to signify if the quote should embed its source (where valid). Then, we don't have to do "attribute only" in `format_quote_text` and can have both the quote and an embed, such as with the Pale Blue Dot.

Also, for the sake of finding future bugs, perhaps we can add some logging to `quote_compliant` so it tells us in which step it failed, so when a ill-formed quote is added it can be tracked down faster. Ideally, more in general, but it's fine for now.

Also, for the VPS' sake, perhaps we move to a rotating file logger, so it's not a storage leak; however slow, can set the size to be quite large, say a couple MB, and 5 or more backups/logs for "I really still want lots of logs kept just in case". `logging` does enable multiple outputs when configured, if wanted. Not too pushy on this, if you're happy with the stdout as is with your existing external logs, we can leave it alone (hence no changes here on this).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,15 +39,19 @@ However, the `attribution` field will appear after the quote, behind a `~`, if g
...
+When citing something said by a character, use a simple rule of thumb: if you know who wrote it, cite as `Author, Character`, otherwise cite the character directly. For example, `The Doctor` or `Jean-Luc Picard` are cited as themselves, because various scriptwriters and actors are involved, whereas Macbeth is cited as `William Shakespeare, Macbeth`, as we're quoting his script (the character should be listed, though in the case of Macbeth, it ould be redundant). Other examples, such as `Sherlock Holmes`, may be context dependent, such as the books will be cited as `Arthur Conan Doyle, Sherlock Holmes`, whereas film or show adaptations are cited accordingly (`Character, Film/Show, Year`). Shows may include the specific episode by name and perhaps number, where relevant, but further details (timestamps etc.) should not be included (but may be part of a `source` URL, such as timestamped links to YouTube).
```
Comment (Gnomeball) [inline README.md:44]: "it ould be redundant", typo.

Comment (AlexBlandin) [reply README.md:44]: Well spotted, I'll correct that momentarily.

Comment (AlexBlandin) [reply README.md]: Corrected

**Comment by Gnomeball** [comment]
I do not feel like we can accept "quotes that are actually just videos" unless there videos are very obviously and very clearly a single stream that can be considered a quote.  What I mean here is a speech, or a poem, or maybe even a short story sure, but anything more cannot reasonably be considered "a quote", but rather "a series of sentences and paragraphs, some of which might be entertaining enough to be quotes"

**Comment by Gnomeball** [comment]
Regarding proposed log changes, I'm all for increased logging.

**Comment by AlexBlandin** [comment]
Yeah, I wasn't sure about it, so really just wanted to test what we thought about longer form "quotes" like that that make use of the embed; given the length of Pale Blue Dot, I wasn't sure quite what we felt was the appropriate cutoff, and where we drew the line at "this is an exceptional, hence rare, long quote".

**Comment by Gnomeball** [comment]
Pale Blue Dot I think is fine because it very clearly is a single stream of text - I liken it to a poem on the state of things - and is very rarely cited as anything but the full text (though often in speeches Sagan would actually extend it).

**Pull request merged**